hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestOmTableInsightTask.java

```diff
@@ -75,8 +79,9 @@ public class TestOmTableInsightTask extends AbstractReconSqlDBTest {
 
   private void initializeInjector() throws IOException {
     reconOMMetadataManager = getTestReconOmMetadataManager(
-        initializeNewOmMetadataManager(temporaryFolder.newFolder()),
-        temporaryFolder.newFolder());
+        initializeNewOmMetadataManager(Files.createDirectory(
+            temporaryFolder.resolve("JunitOmDBDir")).toFile()),
```

Comment:
Could this cause data leakage to other tests? initializeInjector is called before each test where we create a directory with the same name each time.

Reply:
`temporaryFolder` is a `@TempDir`, so the path is different for each test case.  Example:

```
/tmp/junit5823932975861174146/JunitOmDBDir
/tmp/junit842977494714222952/JunitOmDBDir
/tmp/junit2826054961670227457/JunitOmDBDir
/tmp/junit6353237819497228942/JunitOmDBDir
/tmp/junit1432827117522397492/JunitOmDBDir
/tmp/junit4588903889103211047/JunitOmDBDir
/tmp/junit8280824666979319137/JunitOmDBDir
```

Reply:
Thank you Attila. LGTM